egs2/TEMPLATE/asr1/pyscripts/feats/ssl_feature_utils.py

```diff
@@ -41,7 +76,11 @@ def get_feats(self, data, ref_len=None):
 
 
 class MfccFeatureReader(BaseFeatureReader):
-    def __init__(self, sample_rate=16000):
+    def __init__(
+        self,
+        sample_rate=16000,
+        **kwargs,  # placeholder for unused arguments
```

Comment:
As @pengchengguo suggested, I used `kwargs` as a placeholder for unused arguments.